plgrid/plgdata_methods.py

```diff
@@ -0,0 +1,54 @@
+import requests
+
+import tempfile
+
+from pathlib import Path
+
+from plgrid.string_templates import plgdata_get_url, plgdata_list_url
+
+from pymchelper.input_output import fromfile
+
+from yaptide.celery.tasks import pymchelper_output_to_json
+
+http_plgdata = 'https://data.plgrid.pl'
+hostname = 'ares'
```

Comment:
Probably should not be hardcoded in the script

Reply:
PR is still draft -> this will be changed